### PR TITLE
Bump k3s root to v0.12.0 and remove strongswan binaries

### PR DESCRIPTION
**Pull request by brandond**
#### Proposed Changes ####

Bump k3s-root and remove embedded strongswan support. This saves 2124 kB of space from the K3s binary.

before/after K3s binary size comparison:
```
k3s binary dist/artifacts/k3s size 67129344 is less than max acceptable size of 73400320 bytes (70 MiB)
k3s binary dist/artifacts/k3s size 64954368 is less than max acceptable size of 73400320 bytes (70 MiB)
```

#### Types of Changes ####

version bump

#### Verification ####

* Configure a cluster (1 or more nodes) using --flannel-backend=ipsec
* Upgrade the cluster without installing strongswan on the host: note that k3s fails to start
* Upgrade the cluster after installing strongwan packages on the host: note that k3s continues to work as usual

#### Linked Issues ####

* https://github.com/k3s-io/k3s/issues/6027

#### User-Facing Change ####

```release-note
The embedded k3s-root version has been bumped to v0.12.0, based on buildroot 2022.08.1.
The embedded swanctl and charon binaries have been removed. If you are using the ipsec flannel backend, please ensure that the strongswan `swanctl` and `charon` packages are installed on your node before upgrading k3s.
```

### CODE REVIEW

**Comment by cwayne18** [comment]
Please don't merge this until we get signoff from William

**Comment by brandond** [comment]
@cwayne18 just to confirm, we have sign-off on this correct? As long as we document the change?

**Comment by brandond** [comment]
Confirmed via Slack

**Review by dereknola — APPROVED** [review]

**Comment by cwayne18** [comment]
Im fine with this landing, but we just need to make sure the release notes and docs include instructions for getting their own ipsec running

**Review by VestigeJ — APPROVED** [review]
LGTMIO

**Review by briandowns — APPROVED** [review]

**Review by galal-hussein — APPROVED** [review]

**Comment by brandond** [comment]
s390x flaked due to an issue with rdma cgroups not getting cleaned up on the CI node. Merging.

**Pull request merged**
